wiki/Article_Style_Guide/en.md

```diff
@@ -499,29 +568,30 @@ Use of this type of link may include:
 
 ### Link Text
 
-Do not use the word `here` as the link text.
-As it may seem convenient to use, it could be misleading.
-Instead, take the page's title (from the top of page, or from the `<title>` tag) and use that as the link text.
+Do **not** use the word `here` as the link text.
+As it may seem convenient to use, this may cause confusion and/or could be misleading.
+
+Instead, take the page's title (from the top of page or from the `<title/>` tag) and use that as the link text.
```

Comment:
In layman's terms:
- don't do this: `you can get osu! [here](https://osu.ppy.sh/p/download)`
  - I mean, where is "here"? I could hover over the text, but do I _really_ want to do that?
- do this: `you can get osu! at [Download osu!](https://osu.ppy.sh/p/download)`
  - `Download osu!` is part of the `<title/>` tag (and the tab name)

The "top of the page" means from the tab which comes from the `<title/>` tag but those don't display all of the text most of the time :/. It can also mean the text from the first heading of the external page.

tbh, I don't really go into the HTML code either... I just use the console
```js
document.title
```

Despite that code bit above, I made and am using a bookmarklet instead:
```js
javascript:title="[" + document.title + "](" + document.URL + ")";if(title){void(prompt('Page Title',title));}
```
That above gives me the link and the title in markdown so I can just copy that instead. (This was really helpful for when I was fixing the Announcements Archive.)